.gitignore

```diff
@@ -3,6 +3,7 @@ target
 node_modules
 public
 documentation
+/static/
```

Comment:
Is this correct?

Reply:
My bad. For some reason, I had lots of files under /static and thought they had been generated by some build script. I've removed it.